From 701cddcf67faff9234d1c07278f580aea10cd71a Mon Sep 17 00:00:00 2001
From: Awais Belal <awais_belal@mentor.com>
Date: Fri, 11 Jan 2019 17:20:07 +0500
Subject: [PATCH 2/7] init-install-efi: do not blacklist root device for udev

We want all partitions to show up so that SWUpdate can
create its links accordingly.

Signed-off-by: Awais Belal <awais_belal@mentor.com>
---
 init-install-efi.sh | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/init-install-efi.sh b/init-install-efi.sh
index 5fb2ae4..6e713ba 100644
--- a/init-install-efi.sh
+++ b/init-install-efi.sh
@@ -273,10 +273,6 @@ if [ -d /tgt_root/etc/ ] ; then
     swap_part_uuid=$(blkid -o value -s PARTUUID ${swap})
     echo "/dev/disk/by-partuuid/$swap_part_uuid                swap             swap       defaults              0  0" >> /tgt_root/etc/fstab
     echo "UUID=$boot_uuid              /boot            vfat       defaults              1  2" >> /tgt_root/etc/fstab
-    # We dont want udev to mount our root device while we're booting...
-    if [ -d /tgt_root/etc/udev/ ] ; then
-        echo "${device}" >> /tgt_root/etc/udev/mount.blacklist
-    fi
 fi
 
 umount /src_root
-- 
2.11.1

